src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ColorFilter.tsx

```diff
@@ -85,17 +86,24 @@ const ColorFilterOption: React.FC<{ colorOption: ColorOption }> = ({
     </Checkbox>
   )
 }
-
 interface ColorFilterProps {
   expanded?: boolean
 }
 
 export const ColorFilter: React.FC<ColorFilterProps> = ({
   expanded = false,
 }) => {
+  const { currentlySelectedFilters } = useArtworkFilterContext()
+  const showMoreExpanded =
+    intersection(
+      currentlySelectedFilters().colors,
+      COLOR_OPTIONS.slice(INITIAL_ITEMS_TO_SHOW).map(({ value }) => value)
+    ).length > 0
+
+  const hasColorFilter = currentlySelectedFilters().colors.length > 0
   return (
-    <Toggle label="Color" expanded={expanded}>
-      <ShowMore>
+    <Toggle label="Color" expanded={hasColorFilter || expanded}>
```

Comment:
Oh, I was noticing this as well. This would always re-render to have the entire 'Color' section be hidden, (even if 'Show more' had been expanded). So you can force it with `expanded`, but otherwise it will expand if you have _any_ filter in that section selected (which I think makes sense for the behavior in any filter where it's not just always expanded).